react/tic-tac-toe/src/app/screens/Game/components/Square/index.js

```diff
@@ -1,11 +1,28 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 import styles from './styles.module.scss';
 
-const Square = () => (
-  <button type="button" className={styles.square}>
-    {/* TODO */}
-  </button>
-);
+function Square({ value, item, onClick }) {
+  function handleClick() {
+    onClick(item);
+  }
+
+  return (
+    <button
+      type="button"
+      className={styles.square}
+      onClick={handleClick}
+    >
+      {value}
+    </button>
+  );
+}
+
+Square.propTypes = {
+  item: PropTypes.number.isRequired,
+  onClick: PropTypes.func.isRequired,
+  value: PropTypes.string
+};
```

Comment:
Do you need a specific component for just a button? It's just a question, we can talk about it.

Reply:
@LucasZibell  Are you referring to `MoveButton`?